src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -68,6 +68,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Containers.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Keycloak.Tests" />
```

Comment:
We have been trying to avoid adding IVT to extension tests. Is this necessary? If so, why?

Reply:
#4988